modules/promotions/assets/js/editor/module.js

```diff
@@ -21,7 +21,8 @@ export default class Module extends elementorModules.editor.utils.Module {
 	}
 
 	registerControlBehavior( behaviors, view ) {
-		if ( 'display_conditions_pro' !== view.options.model.get( 'name' ) ) {
+		const controlsToSkip = [ 'display_conditions_pro', 'scrolling_effects_pro', 'mouse_effects_pro', 'sticky_pro' ];
+		if ( ! controlsToSkip.includes( view.options.model.get( 'name' ) ) ) {
```

Comment:
```suggestion
		const promotionsControls = [ 'display_conditions_pro', 'scrolling_effects_pro', 'mouse_effects_pro', 'sticky_pro' ];
		if ( ! promotionsControls.includes( view.options.model.get( 'name' ) ) ) {
```
Maybe something like this? It is not controls to skip, it's controls that we want to run the logic for.

Reply:
Fixed